src/ncp/ncp_base.cpp

```diff
@@ -3613,6 +3618,47 @@ otError NcpBase::GetPropertyHandler_NCP_CNTR(uint8_t header, spinel_prop_key_t k
     return errorCode;
 }
 
+otError NcpBase::GetPropertyHandler_IP_CNTR(uint8_t header, spinel_prop_key_t key)
+{
+    uint32_t value;
+    otError errorCode = OT_ERROR_NONE;
+
+    switch (key)
+    {
+    case SPINEL_PROP_CNTR_IP_TX_SUCCESS:
+        value = mInstance->mThreadNetif.GetMeshForwarder().GetCounters().mTxSuccess;
```

Comment:
Expose a public API to retrieve the counters, rather than reaching directly into the instance?